test/e2e/publicviewer/proxy_test.go

```diff
@@ -0,0 +1,181 @@
+package publicviewer_test
```

Comment:
as mentioned in the comment for the `test.mk` file maybe you don't need to have a separate test suite, you could move this under the test/e2e folder ( not the parallel one maybe ), and update the toolchainconfig at runtime.

Reply:
done in c4660e0332eb71ee3bec7645110ea5df8869ccb9